README.md

```diff
@@ -1,23 +1,50 @@
 # Node Feature Discovery Operator
 
-The Node Feature Discovery operator manages detection of hardware features and configuration in a kubernetes cluster.
-The operator orchestrates all resources needed to run the [Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) DaemonSet
+The Node Feature Discovery operator is a tool for Kubernetes administrators that makes it easy to detect and understand the hardware features and configurations of a cluster's nodes. With this operator, administrators can easily gather information about their nodes that can be used for scheduling, resource management, and more by controlling the life cycle of the [Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) DaemonSet.
 
-Welcome to Node Feature Discovery Operator!
+## How it Works
 
-### See our [Documentation][documentation] for detailed instructions and reference
+The operator works by orchestrating all resources needed to run the Node-Feature-Discovery DaemonSet. This DaemonSet runs on each node in the cluster and detects the features and configurations of the node's hardware. The operator then collects and organizes this information, making it easily accessible.
 
-## Community, discussion, contribution, and support
+## Quick start
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+Get the source code
```

Comment:
Do we want to advertise this as the preferred deployment method? How about just

```
kubectl create -f https://operatorhub.io/install/nfd-operator.yaml
```

Reply:
this is the same `Quick start` from the documentation pages, that way you point at requires an extra step to enable the operatorhub on the user cluster.

I am basically bringing the quick start section from the documentation to the readme for a "quick start"

Reply:
that's where I am trying to move, I want to have HELM charts for the operator, been busy with other stuff, but I am planning to add HELM soon